gluster.yml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
I would like to clearly distinguish between playbooks which deals with gluster components only (eg. `gluster_peer_bricks.yml` and those which handles both gluster and tendrl).